rpc/wrtc_call_queue_mongodb.go

```diff
@@ -889,7 +885,7 @@ func (queue *mongoDBWebRTCCallQueue) SendOfferInit(
 	sendCtx, sendCtxCancel := context.WithDeadline(ctx, offerDeadline)
 
 	// need to watch before insertion to avoid a race
-	sendAndQueueCtx, sendAndQueueCtxCancel := utils.MergeContext(sendCtx, queue.cancelCtx)
+	sendAndQueueCtx, sendAndQueueCtxCancel := utils.MergeContext(sendCtx, queue.activeStoppableWorkers.Context())
```

Comment:
I suppose we can keep this to minimize the changes as part of this refactor. But this feels clunky. We're just merging here to reduce the number of cases inside the below selects by one.

And I think there are only two select statements.